Source/WebCore/html/HTMLAttachmentElement.h

```diff
@@ -119,9 +122,11 @@ class HTMLAttachmentElement final : public HTMLElement {
     FloatSize m_iconSize;
 
     RefPtr<HTMLAttachmentElement> m_innerLegacyAttachment;
+    RefPtr<HTMLElement> m_elementContainer;
     RefPtr<HTMLElement> m_elementWithAction;
     RefPtr<HTMLElement> m_elementWithTitle;
     RefPtr<HTMLElement> m_elementWithSubtitle;
+    RefPtr<HTMLElement> m_elementWithSave;
```

Comment:
I should have probably brought this up before, but I preferred if we named these differently:

```suggestion
    RefPtr<HTMLAttachmentElement> m_attachmentIcon;
    RefPtr<HTMLElement> m_containerElement;
    RefPtr<HTMLElement> m_actionTextElement;
    RefPtr<HTMLElement> m_titleElement;
    RefPtr<HTMLElement> m_subtitleElement;
    RefPtr<HTMLElement> m_saveButton;
```

Reply:
Good idea, thanks.
Though I'll keep `m_innerLegacyAttachment` for now (because it currently does more than just display an icon) but I'll add a mention in the FIXME about changing the name when we'll get rid of the inner-attachment trick.